src/components/numberinput/index.js

```diff
@@ -0,0 +1,76 @@
+import { Lightning } from 'wpe-lightning-sdk'
+import Config from './config.js'
+import * as player from '@/services/player/'
+import { channelsServiceInit, getCurrentProgram, getChannel } from '@/services/ChannelsService'
+import { ChannelNumber } from './../../channelnumber.js'
+
+export default class NumberInput extends Lightning.Component {
```

Comment:
Please follow the same naming convention as for other components so instead of numberinput folder use NumberInput

Reply:
numberinput  folder is renamed  to "NumberInput"